host/src/pack/dpkg/etc/civ.sh

```diff
@@ -11,6 +11,7 @@ do
 done
 
 /bin/sh -e /etc/rc.civ start
+/bin/bash /etc/sharefolder_monitor.sh &
```

Comment:
In PR 28, the civ service has been modified for service graceful shutdown and civ.sh is no longer used. This script may need to be added into rc.civ.post.